discovery-provider/plugins/notifications/src/processNotifications/mappers/cosign.ts

```diff
@@ -60,23 +60,16 @@ export class CosignRemix extends BaseNotification<CosignRemixNotificationRow> {
         await Promise.all(devices.map(device => {
           return sendPushNotification({
             type: device.type,
-            badgeCount: userNotifications.mobile[this.parentTrackUserId].badgeCount,
+            badgeCount: userNotifications.mobile[this.parentTrackUserId].badgeCount + 1,
             targetARN: device.awsARN
           }, {
             title: 'New Track Co-Sign! 🔥',
             body: `${parentTrackUserName} Co-Signed your Remix of ${remixTrackTitle}`,
             data: {}
           })
         }))
-        // TODO: increment badge count
+        await this.incrementBadgeCount(this.parentTrackId)
```

Comment:
blocker - is this supposed to be `parentTrackUserId`?

Reply:
could we add to the tests verification that badge count is incremented in the db or that this `incrementBadgeCount` is called? looks like it's easy to miss an error here in the tests

Reply:
Thanks for catching that
There are updated tests checking that badgecount is 1 and in the `multipleMappings.test.ts` that it was incremented twice to two

Reply:
![Screenshot 2023-03-14 at 10 48 56 AM](https://user-images.githubusercontent.com/7064122/225038990-762b59c4-9c91-4e1c-b20a-3c9e564005cb.png)
There no need to validate mock call, in this code we just validate that the expected change from this method actually changes the db value